apps/studio/src/utils/resources.ts

```diff
@@ -0,0 +1,10 @@
+import { ResourceType } from "@prisma/client"
+
+export const isAllowedToHaveChildren = (
+  resourceType: ResourceType,
+): boolean => {
+  return (
+    resourceType !== ResourceType.Page &&
```

Comment:
nit: usually for this, i would prefer a whitelist approach than blacklist checking, so that even if we extend to have more resource types, our condition is explicit and clear

Reply:
sgtm